packages/connection-form/src/components/advanced-options-tabs/authentication-tab/authentication-oidc.tsx

```diff
@@ -16,6 +17,8 @@ import { errorMessageByFieldName } from '../../../utils/validation';
 import { getConnectionStringUsername } from '../../../utils/connection-string-helpers';
 import type { OIDCOptions } from '../../../utils/oidc-handler';
 import { useConnectionFormPreference } from '../../../hooks/use-connect-form-preferences';
+import { usePreference } from 'compass-preferences-model/provider';
+import { useGlobalAppRegistry } from 'hadron-app-registry';
```

Comment:
This is less of a problem than using preferences directly, but just from the perspective of the connection-form being a component that is not exactly aware of compass, we might want to have just a normal callback prop like `onOpenProxySettingsClick` passed down from the connection form root instead of emitting on global app registry directly in the form code. Would changing this up make sense to you?

Reply:
Yup, done in 42c22a6e51f217ed4e73c57b19911be38c0ec385 👍